github/actions_workflow_runs.go

```diff
@@ -14,33 +14,36 @@ import (
 
 // WorkflowRun represents a repository action workflow run.
 type WorkflowRun struct {
-	ID               *int64         `json:"id,omitempty"`
-	Name             *string        `json:"name,omitempty"`
-	NodeID           *string        `json:"node_id,omitempty"`
-	HeadBranch       *string        `json:"head_branch,omitempty"`
-	HeadSHA          *string        `json:"head_sha,omitempty"`
-	RunNumber        *int           `json:"run_number,omitempty"`
-	Event            *string        `json:"event,omitempty"`
-	Status           *string        `json:"status,omitempty"`
-	Conclusion       *string        `json:"conclusion,omitempty"`
-	WorkflowID       *int64         `json:"workflow_id,omitempty"`
-	CheckSuiteID     *int64         `json:"check_suite_id,omitempty"`
-	CheckSuiteNodeID *string        `json:"check_suite_node_id,omitempty"`
-	URL              *string        `json:"url,omitempty"`
-	HTMLURL          *string        `json:"html_url,omitempty"`
-	PullRequests     []*PullRequest `json:"pull_requests,omitempty"`
-	CreatedAt        *Timestamp     `json:"created_at,omitempty"`
-	UpdatedAt        *Timestamp     `json:"updated_at,omitempty"`
-	JobsURL          *string        `json:"jobs_url,omitempty"`
-	LogsURL          *string        `json:"logs_url,omitempty"`
-	CheckSuiteURL    *string        `json:"check_suite_url,omitempty"`
-	ArtifactsURL     *string        `json:"artifacts_url,omitempty"`
-	CancelURL        *string        `json:"cancel_url,omitempty"`
-	RerunURL         *string        `json:"rerun_url,omitempty"`
-	HeadCommit       *HeadCommit    `json:"head_commit,omitempty"`
-	WorkflowURL      *string        `json:"workflow_url,omitempty"`
-	Repository       *Repository    `json:"repository,omitempty"`
-	HeadRepository   *Repository    `json:"head_repository,omitempty"`
+	ID                 *int64         `json:"id,omitempty"`
+	Name               *string        `json:"name,omitempty"`
+	NodeID             *string        `json:"node_id,omitempty"`
+	HeadBranch         *string        `json:"head_branch,omitempty"`
+	HeadSHA            *string        `json:"head_sha,omitempty"`
+	RunNumber          *int           `json:"run_number,omitempty"`
+	RunAttempt         *int           `json:"run_attempt,omitempty"`
```

Comment:
Please revert this file... this somehow got merged in from another PR and should not be in this one.